### PR TITLE
try to improve the "topological models" intro

### LINKED ISSUE #231: reduce verbosity in Section 5
The three sentences _"There are multiple [...]"_, _"This section includes [...]"_ and _"This is not intended [...]"_ are slightly vague.  It doesn't seem to me that their information content is enough to justify them being independent sentences.
They should probably be combined.

### PR DESCRIPTION
**Pull request by thomas-fossati**
Fixes #231

### CODE REVIEW

File: draft-ietf-rats-architecture.md [hunk 1]
```diff
@@ -607,12 +607,13 @@ Produced By: Reference Value Provider.
...
+{{dataflow}} shows an abstract model for communication between an Attester,
 a Verifier, and a Relying Party. The Attester conveys its Evidence to the Verifier
 for appraisal, and the Relying Party gets the Attestation Result from the Verifier.
-There are multiple other possible models. This section includes some reference
-models. This is not intended to be a restrictive list, and other variations may
-exist.
+This section refines it by describing two reference interaction schemes
+(topological models), as well as one possible compositions thereof. The discussion
+that follows is for illustrative purposes only and does not constrain the
+interactions between RATS roles to the presented patterns.
```
Comment (mcr) [inline draft-ietf-rats-architecture.md:616]:
```suggestion
This section refines the model by describing two topologies as reference interaction schemes. 
In addition, there is a composition of the models possible.
The discussion that follows is for illustrative purposes only and does not constrain the
interactions between RATS roles to the presented patterns.
```

**Review by mcr — APPROVED** [review]

File: draft-ietf-rats-architecture.md [hunk 2]
```diff
@@ -607,12 +607,13 @@ Produced By: Reference Value Provider.
 
 # Topological Models {#overview}
 
-{{dataflow}} shows a basic model for communication between an Attester,
+{{dataflow}} shows an abstract model for communication between an Attester,
```
Comment (William-PanWei) [inline draft-ietf-rats-architecture.md:610]: Is it necessary to change "basic" to "abstract"?
"An abstract model" makes me feel that this model is not practical. But actually, it is a viable model, so I think it's a basic model.

Comment (dthaler) [inline draft-ietf-rats-architecture.md:610]:
```suggestion
{{dataflow}} shows data-flow diagram for communication between an Attester,
```

File: draft-ietf-rats-architecture.md [hunk 3]
```diff
@@ -607,12 +607,13 @@ Produced By: Reference Value Provider.
...
 a Verifier, and a Relying Party. The Attester conveys its Evidence to the Verifier
 for appraisal, and the Relying Party gets the Attestation Result from the Verifier.
-There are multiple other possible models. This section includes some reference
-models. This is not intended to be a restrictive list, and other variations may
-exist.
+This section refines it by describing two reference interaction schemes
```
Comment (dthaler) [inline draft-ietf-rats-architecture.md:613]:
```suggestion
This section refines it by describing two reference interaction models
```

File: draft-ietf-rats-architecture.md [hunk 4]
```diff
@@ -607,12 +607,13 @@ Produced By: Reference Value Provider.
...
+(topological models), as well as one possible compositions thereof. The discussion
```
Comment (dthaler) [inline draft-ietf-rats-architecture.md:614]:
```suggestion
(topological models), as well as one example composition thereof. The discussion
```

File: draft-ietf-rats-architecture.md [hunk 5]
```diff
@@ -607,12 +607,13 @@ Produced By: Reference Value Provider.
 
 # Topological Models {#overview}
```
Comment (mcr) [inline draft-ietf-rats-architecture.md:608]:
```suggestion
# Topological Patterns {#overview}
```

**Pull request merged**